sdk/tests/conformance/extensions/ext-disjoint-timer-query.html

```diff
@@ -241,18 +241,22 @@
 }
 
 function checkQueryResults() {
-    if (availability_retry > 0 && (timestamp_counter_bits == 0 ||
-        !ext.getQueryObjectEXT(timestamp_query2, ext.QUERY_RESULT_AVAILABLE_EXT))) {
-        var error = gl.getError();
-        if (error != gl.NO_ERROR) {
-            testFailed("getQueryObjectEXT should have no errors: " + wtu.glEnumToString(gl, error));
-            debug("");
-            finishTest();
+    if (availability_retry > 0) {
+        // Make a reasonable attempt to wait for the queries' results to become available.
+        if (!ext.getQueryObjectEXT(elapsed_query, ext.QUERY_RESULT_AVAILABLE_EXT) ||
+            (timestamp_counter_bits > 0 && !ext.getQueryObjectEXT(timestamp_query2, ext.QUERY_RESULT_AVAILABLE_EXT))) {
+            var error = gl.getError();
+            if (error != gl.NO_ERROR) {
+                testFailed("getQueryObjectEXT should have no errors: " + wtu.glEnumToString(gl, error));
+                debug("");
+                finishTest();
+                return;
+            }
+            availability_retry--;
+            gl.finish();
```

Comment:
Actually, the gl.finish() is not really relevant. I can remove it if it is causing confusion.

The reason the test runs much faster is that the if-test above was wrong. timestamp_counter_bits is always 0 on macOS, and I think, actually 0 on all platforms Chrome runs on. This was causing this part of the test to always count down availability_retry from 512 to 0 before proceeding.

Please let me know if you'd rather I remove the gl.finish() call.